network/p2p/acp118/handler_test.go

```diff
@@ -131,6 +128,8 @@ func TestHandler(t *testing.T) {
 				if ok {
 					require.Equal(sig, response.Signature)
 				}
+
+				require.Equal(serverNodeID, nodeID)
```

Comment:
nit: We don't need to check for this IMO because this is guaranteed to match the responding peer from `p2p` and we already have tests on this in the upstream package - even if this line broke it wouldn't be possible to fix it from the perspective of acp118. If we did want to keep this... we should assert this after checking for a non-nil appErr before we check the response on L118 IMO.

Reply:
I'm not really picky on this one, so I'll just omit it for now.